packages/vite-plugin/tests/out/with-sourcemaps/__snapshots__/build.test.ts.snap

```diff
@@ -41,49 +41,31 @@ Object {
 
 exports[`build fs output > _01 output files 1`] = `
 Array [
-  "assets/background.ts.hash2.js",
-  "assets/content.ts-loader.hash0.js",
-  "assets/content.ts.hash1.js",
-  "assets/popup.html.hash3.js",
-  "assets/vendor.hash4.js",
+  "assets/background.ts.hash1.js",
+  "assets/content.ts.hash0.js",
+  "assets/popup.html.hash2.js",
+  "assets/vendor.hash3.js",
   "manifest.json",
   "service-worker-loader.js",
   "src/popup.html",
 ]
 `;
 
-exports[`build fs output > assets/background.ts.hash2.js 1`] = `
+exports[`build fs output > assets/background.ts.hash1.js 1`] = `
 "console.log(\\"service_worker.ts\\");
 // # sourceMappingURL=data:application/json;charset=utf-8;base64,<base64>
 "
 `;
 
-exports[`build fs output > assets/content.ts.hash1.js 1`] = `
-"const message = \\"content script\\";
+exports[`build fs output > assets/content.ts.hash0.js 1`] = `
+"(function(){const message = \\"content script\\";
 console.log(message);
-// # sourceMappingURL=data:application/json;charset=utf-8;base64,<base64>
-"
-`;
-
-exports[`build fs output > assets/content.ts-loader.hash0.js 1`] = `
-"(function () {
-  'use strict';
-
-  const injectTime = performance.now();
-  (async () => {
-    const { onExecute } = await import(
-      /* @vite-ignore */
-      chrome.runtime.getURL(\\"assets/content.ts.hash1.js\\")
-    );
-    onExecute?.({ perf: { injectTime, loadTime: performance.now() - injectTime } });
-  })().catch(console.error);
-
-})();
+})()//# sourceMappingURL=data:application/json;charset=utf-8;base64,eyJ2ZXJzaW9uIjozLCJmaWxlIjoiY29udGVudC50cy5oYXNoNWFmYjk4MDAuanMiLCJzb3VyY2VzIjpbIi4uLy4uL3NyYy9jb250ZW50LnRzIl0sInNvdXJjZXNDb250ZW50IjpbInR5cGUgTG9nID0gJ2NvbnRlbnQgc2NyaXB0J1xuY29uc3QgbWVzc2FnZTogTG9nID0gJ2NvbnRlbnQgc2NyaXB0J1xuY29uc29sZS5sb2cobWVzc2FnZSlcbmV4cG9ydCB7fVxuIl0sIm5hbWVzIjpbXSwibWFwcGluZ3MiOiJBQUNBLE1BQU0sVUFBZTtBQUNyQixRQUFRLElBQUksT0FBTzsifQ==
```

Comment:
how is that different. Looks like outdated tests on main

Reply:
same issue as https://github.com/crxjs/chrome-extension-tools/pull/997#discussion_r2030621822 - fixed with latest force-push